src/main/java/org/broadinstitute/hellbender/tools/picard/sam/BamToBfq.java

```diff
@@ -0,0 +1,492 @@
+package org.broadinstitute.hellbender.tools.picard.sam;
```

Comment:
I think this should be deleted.  Must have been deleted in master and re-added by accident.